lms/static/js/fixtures/accordion.html

```diff
@@ -8,21 +8,21 @@
     <div class="chapter-content-container is-open" id="accordion-menu-1" tabindex="-1">
       <div class="chapter-menu is-open">
           <div class="menu-item">
-          <a href="#">
+          <button type="button">
```

Comment:
Yes. Not sure why this fixture wasn't updated like a year ago. Surprised tests pass...